src/components/FileUploader/FileUploader.js

```diff
@@ -130,7 +130,7 @@ export class FileUploaderButton extends Component {
       ...other
     } = this.props;
     const classes = classNames({
-      'bx--file': true,
+      'bx--btn': true,
```

Comment:
Would you want to add `bx--btn--${buttonKind}` here so you don't have to do ``className={`bx--btn--${buttonKind} ${classes}`}``?